### ISSUE TITLE
Bower installs to folder "Bootstrap Markdown Editor"

### ISSUE THREAD

**Issue by mlmedia** [inacho/bootstrap-markdown-editor#23]
"bower install --save bootstrap-markdown-editor"
...installs to a "Bootstrap Markdown Editor" directory

Can we make the default directory "bootstrap-markdown-editor" (without spaces) which would conform to other bower component installations?

Probably not worth going through a pull request as it should be a fairly simple change for the repo owner.

NOTE: I have a custom Bower installation directory with a .bowerrc file:

{
"directory":"public/src/components"
}

Maybe this is the reason?

**Comment by djadomi** [comment]
No, your custom Bower installation directory isn't the reason; it's the same for me. Bower complains about the fact on install. I just symlinked to the directory - bootstrap-markdown-editor was the path injected into my index.html.

**Comment by Schlurcher** [comment]
Package name is not adhering to specs.

https://github.com/bower/spec/blob/master/json.md#name
